functions/update/index.js

```diff
@@ -0,0 +1,40 @@
+import { parseAssignedProperties, fetchRecord } from '../utils';
+
+const update = async ({
+  selectedRecord: {
+    data: { id: recordId },
```

Comment:
Maybe we could rename `recordId` to `id` as we no longer destructure the data returned from the DataAPI to get the record id.